lib/execution/transaction.js

```diff
@@ -93,6 +115,18 @@ class Transaction extends EventEmitter {
     return this.query(conn, `RELEASE SAVEPOINT ${this.txid};`, 1, value);
   }
 
+  setIsolationLevel(isolationLevel) {
```

Comment:
`transaction()` method already has optional `config` parameter, why not just add isolation level there?

Reply:
Ummm, I could? It's basically the same, but I've never seen any documentation for the config param and I thought the chaining style was more idiomatic 🤷

Reply:
Yeah, I'll add types and documentation for the config param. And currently there is no overload for .transaxtion with just a config param, I think.